packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2304,7 +2328,7 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
   ///
   /// This property is typically used to notify the renderer of input gestures
   /// when [RenderEditable.ignorePointer] is true.
-  RenderEditable? get renderEditable => _editableKey.currentContext!.findRenderObject() as RenderEditable?;
+  RenderEditable get renderEditable => _editableKey.currentContext!.findRenderObject()! as RenderEditable;
```

Comment:
Nice way of getting rid of all of the exclamation marks elsewhere in the file.  I'm still figuring out all of the NNBD stuff but it's looking pretty slick.